styles/styles.css

```diff
@@ -1,54 +1,57 @@
 @import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Russo+One&display=swap");
 
 * {
-  font-family: "Roboto", sans-serif;
-  box-sizing: border-box;
+    font-family: "Roboto", sans-serif;
+    box-sizing: border-box;
 }
 .container {
-  background-color: #ffffff;
-  width: 1440px;
-  height: 570px;
-  padding-top: 120px;
+    background-color: #ffffff;
+    width: 1440px;
+    height: 570px;
 }
+
 .offer {
-  font-weight: 500;
-  font-size: 40px;
-  height: 47px;
-  margin: 0 567px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    height: 189px;
+    padding-top: 120px;
+}
+.offer-title {
+    font-weight: 500;
+    font-size: 40px;
+    height: 47px;
+    margin: 0 567px;
+    width: 306px;
 }
 
-.line {
-  border: 2px solid #f5bd41;
-  background-color: #f5bd41;
-  width: 100px;
-  margin-top: 20px;
+.offer-line {
+    height: 4px;
+    width: 100px;
+    margin: 18px 670px 0 670px;
+    background-color: #f5bd41;
 }
 
-.container-1 {
-  display: flex;
-  margin: 84px 139px 122px 181px;
-  align-items: center;
-  justify-content: space-between;
+.advantages {
+    display: flex;
+    margin-left: 181px;
```

Comment:
you can add margin into 1 string and with auto parameter for sides - it`s better